### PR TITLE
Add CSS styleguide

### PR DESCRIPTION
**Pull request by MaxWofford**
~~**Do not merge**~~

~~This pr is just a first draft meant for discussing.~~

@hackclub/staff Would love to get your thoughts on this. Please point our any guidelines that you feel should be added/dropped.

EDIT: Removed 'do not merge' lines

### CODE REVIEW

**Comment by MaxWofford** [comment]
Actually, I take back what I said about this PR being just for discussion; let's seriously consider it and then submit PRs to fix anything that isn't right in it.

**Comment by jonleung** [comment]
I am down for these style conventions! Is this document linked to from anywhere? I think it might be worthwhile to link it from the root `CONTRIBUTING.md`

**Comment by MaxWofford** [comment]
@jonleung I'd actually like to store all the styleguides in one folder. I'll submit another PR for that once this and https://github.com/hackclub/meta/pull/398 are merged

**Comment by zachlatta** [comment]
It looks like bolding is being used as headers throughout the document. Is that intentional?

**Comment by MaxWofford** [comment]
@zachlatta Yep! https://github.com/hackclub/meta/pull/398/files#diff-e61e82e39566a33868bcf4f104f1d800R80

**Comment by paked** [comment]
@MaxWofford I think these could potentially be headers.

File: css_style_guide.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+stay consistent.
+
+**Capitalization** Use only lowercase.
+
+**80 characters wide** Where possible, limit lines to 80 characters wide.
+
+**Indentation** CSS should use 2 spaces per level of indentation.
+
+**Multi-line CSS**
+CSS rules should always be multipule lines to reduce merge conflicts.
```
Comment (paked) [inline css_style_guide.md]: > multiple

*typo alert*

**Pull request merged**